numba/cuda/codegen.py

```diff
@@ -11,6 +11,12 @@
 
 
 class CUDACodeLibrary(CodeLibrary):
```

Comment:
Note to self: needs some commentary on the differences between it and other CodeLibrary implementations, in particular how it only handles IR from llvmlite.ir, not entities from llvmlite.binding.